presto-generation-annotation/src/main/java/com/facebook/presto/test/ConnectorFeature.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.test;
+
+public enum ConnectorFeature
```

Comment:
Will an enum do? A connector can lack support for certain data types (this can be enum-ed), or have limited supported for others (like varchar max length). Maybe this should be modelled as feature interface where most impls are an enum (like `java.nio.file.OpenOption`).

Of course, this would require more elaborated support in the annotations, so... i'm myself not sure.